### PR TITLE
Hide generic channel display name and avatar on watch view

### LINKED ISSUE #2953: Default ABC  channel / Main ABC channel
**What happened?**
Below the videos, peertube currently shows words "main" and "default" as a prefix to the channel name. 

**What do you expect to happen instead?**
Is this necessary? Can we use the words as a suffix instead because if I say check out ABC channel or I see a video, its normal to check the icon/name and I find the prefix off.
Maybe im being unreasonable,  question, is this because the user hasnt named their channels?

https://tilvids.com/videos/watch/3b4be045-bcd0-4968-a4e7-99fbde8bd281

This link says
"Main arcaneengine channel".
Can we rewrite this as 
"Arcaneengine - Main Channel"

If we really have to differentiate between channels not based on name but also on having prefix/suffix of the word channel  ?

https://peertube.cpy.re/videos/watch/da2b08d4-a242-4170-b32a-4ec8cbdca701

Here it says
"Default Chocobozzz channel"

### PR DESCRIPTION
**Pull request by kimsible**
First stab at : https://github.com/Chocobozzz/PeerTube/issues/2987
fixes #2953

- No avatar channel and generic display name (Main *username* channel) channel :
*the big avatar and account name redirect to the account page*
no-avatar-and-channel

- Generic display name (Main *username* channel) and a channel avatar is uploaded
*the small avatar is the channel avatar and redirect to the channel page*
no-channel-avatar-not-null

- Display name channel and avatar channel customized
*back to regular display*
n-avatar-and-channel

### CODE REVIEW

File: client/src/app/+videos/+video-watch/video-watch.component.ts [hunk 1]
```diff
@@ -307,6 +307,12 @@ export class VideoWatchComponent implements OnInit, OnDestroy {
     )
   }
 
+  isChannelDisplayNameGeneric () {
+    const genericChannelDisplayName = `Main ${this.video.channel.ownerAccount.name} channel`
```
Comment (Chocobozzz) [inline client/src/app/+videos/+video-watch/video-watch.component.ts:311]: At the beginning of peertube it was also `Default ... channel`. Please add this condition too (or use an array)

**Comment by Chocobozzz** [comment]
Nice job :+1: Just one little thing and we can merge this

**Comment by kimsible** [comment]
> Nice job +1 Just one little thing and we can merge this

Thanks for the review =). I forgot the search result, should we also adapt display for this view ?

image

**Comment by Chocobozzz** [comment]
> I forgot the search result, should we also adapt display for this view ?

I don't think so, because we are explicitly searching for channels. To not confuse users, let's keep the current display in the search page

**Pull request merged**